### ISSUE TITLE
Feature Request: Batch-Mode im Installationsskript

### ISSUE THREAD

**Issue by saschaszott** [OPUS4/application#14]
Ich würde gern eine automatische Installation von OPUS durchführen, bei der ich während der Installation keine Benutzereingaben tätigen muss. Dazu sollte das Installationsskript mit einer zusätzlichen Option `-b` oder `--batch` versehen werden. Die erforderlichen Benutzereingaben (z.B. Passwort für den MySQL-User `root`) sollten per Umgebungsvariablen oder alternativ über weitere Optionen beim Aufruf übergeben werden können. Da sich aber für die meisten Parameter sinnvolle Default-Werte ergeben (die das Skript sogar jetzt schon vorschlägt), ist die Menge der unbedingt erforderlichen Benutzereingaben sicher beschränkt.

**Comment by j3nsch** [comment]
Ja, macht Sinn. Wird in der Entwicklung beim KOBV in 2016 nicht mehr passieren, da die Zeit und Dringlichkeit fehlt.
